core/src/services/iframe.js

```diff
@@ -34,11 +34,15 @@ class IframeClass {
     return container && container.length > 0 ? container[0] : undefined;
   }
 
-  getAllIframes(modalIframe) {
+  getAllIframes(iframeTypeArr = []) {
     const iframes = Array.from(
       document.querySelectorAll('.iframeContainer iframe')
     );
-    if (modalIframe) iframes.push(modalIframe);
+    iframeTypeArr
```

Comment:
What about any of the following variants? 
```
return iframes.concat(iframeTypeArr.filter(Boolean));
``` 
or 
``` 
iframes.push(...iframeTypeArr.filter(Boolean));
return iframes;
```

Reply:
done